docs/README.md

```diff
@@ -0,0 +1,118 @@
+# SHARK User Guide
+
+> [!WARNING]
+> This is still pre-release so the artifacts listed here may be broken
+>
+
+These instructions cover the usage of the latest stable release of SHARK. For a more bleeding edge release please install the [nightly releases](nightly_releases.md).
+
+## Prerequisites
+
+Our current user guide requires that you have:
+- Access to a computer with an installed AMD Instinct™ MI300x Series Accelerator
+- Installed a compatible version of Linux and ROCm on the computer (see the [ROCm compatability matrix](https://rocm.docs.amd.com/en/latest/compatibility/compatibility-matrix.html))
+
+
+## Set up Environment
+
+You will need a recent version of Python. We recommend also setting up a Python environment.
+
+Officially we support Python versions: 3.11, 3.12, 3.13, 3.13t
+
+The rest of this guide assumes you are using Python 3.11.
+
+### Install Python 
+To install Python 3.11 on Ubuntu:
+
+```bash
+sudo apt install python3.11 python3.11-dev python3.11-venv
+
+which python3.11
+# /usr/bin/python3.11
+```
+
+### Create a Python Environment
+
+This guide assumes you'll be using pyenv. Setup your pyenv with the following commands:
+
+```bash
+# Set up a virtual environment to isolate packages from other envs.
+python3.11 -m venv 3.11.venv
+source 3.11.venv/bin/activate
+```
```

Comment:
Hah. Python.

`pyenv` and Python `venv` are two different things. They can work together though.

https://github.com/pyenv/pyenv

If you use `pyenv`, you can get behavior like this:
```bash
python --version
# 3.10

pyenv install 3.11
pyenv global 3.11
python --version
# 3.11
```

Then when you create a venv:
```bash
python -m venv .venv
source .venv/bin/activate
python --version
# 3.11
```

Without venv, you would have the code you have here:
```bash
python --version
# 3.10
python -m venv .venv
source .venv/bin/activate
python --version
# 3.10
deactivate

python3.11 -m venv 3.11.venv
source 3.11.venv/bin/activate
python --version
# 3.11
```

So pyenv lets you just call `python` as if it was `python3.11`, but `venv` is what actually isolates installed packages into a virtual environment